code/modules/admin/topic.dm

```diff
@@ -10,6 +10,85 @@
 		check_antagonists()
 		return
 
+	if(href_list["autoresponse"]) // new verb on the Ahelp.  Will tell the person their message was received, and they probably won't get a response
+		var/mob/ref_person = locate(href_list["autoresponse"])
+		if(!ref_person || !istype(ref_person) || !ref_person.client)
+			to_chat(usr, "\blue Looks like that person stopped existing!")
```

Comment:
Don't use byond color macros

Use the various SPAN_XXX macros, or the span(class, text) proc

In this case, SPAN_NOTICE would be best

Reply:
this is a big block of commented out stuff, please delete it